File: OneDrive/Documents/PhaseGateOne/Fire Drill/Python/task_seven.py
sum1 = 0
sum2 = 0
for index in range(1, 11):
	if index % 4 == 0:
		for count in range(1, 6):
			if (index % 4 == 0):
				if (index == 4):
					sum1 += (index)**count;
				elif index == 8:
					sum2 += (index)**count;

print(sum1 + " " + sum2)
	

File: OneDrive/Documents/PhaseGateOne/Fire Drill/JavaScript/TaskSeven.js
sum1 = 0;
sum2 = 0;

for (let index = 1; index <= 10; index++){
	if (index % 4 == 0){
		if (index == 4){
			for (let count = 1; count <= 5; count++){
			sum1 += Math.pow(index, count);
			}
		} else {
			for (let count = 1; count <= 5; count++){
			sum2 = Math.pow(index, count);
			}
		};
	}
	};

console.log(sum1, " ", sum2)